tests/unit/cli/admin/test_import_targets.py

```diff
@@ -140,80 +176,81 @@ def test__import_csv_to_rstuf_duplicate_targets(self):
         with pytest.raises(import_targets.click.ClickException) as err:
             import_targets._import_csv_to_rstuf(
                 fake_db_client,
-                fake_rstuf_table,
+                fake_rstuf_files,
+                "fake_rstuf_roles",
                 ["csv1", "csv2"],
                 "fake_succinct_roles",
             )
-
         assert "ABORTED due duplicated targets." in str(err)
         assert import_targets._parse_csv_data.calls == [
-            pretend.call("csv1", "fake_succinct_roles"),
+            pretend.call(
+                fake_db_client,
+                "fake_rstuf_roles",
+                "fake_succinct_roles",
+                "csv1",
+            ),
         ]
 
-    def test__get_succinct_roles(self, monkeypatch):
-        import_targets.request_server = pretend.call_recorder(
-            lambda *a: pretend.stub(status_code=200, text=b"data")
+    def test__get_succinct_roles(self):
+        fake_response = pretend.stub(
+            status_code=200,
+            json=pretend.call_recorder(
+                lambda: {"data": {"number_of_delegated_bins": 16}}
+            ),
         )
-        monkeypatch.setattr(
-            import_targets.json, "loads", lambda *a: "json_data"
+        import_targets.request_server = pretend.call_recorder(
```

Comment:
Maybe it's not a good idea to pretend directly the `import_targets` and instead use `monkepatch`?
I recently had problems with functions still being `pretend` after the test execution.

Reply:
It is required only if `import_target` is used in another scope.
In the case of the CLI, used by multiple command lines namespaces, like your case.